src/seashore/tests/test_shell.py

```diff
@@ -114,6 +115,18 @@ def test_repr(self):
         self.assertIn('myout', stringified)
         self.assertIn('myerr', stringified)
 
+    def test_trace(self):
+        """traceback has args"""
+        try:
+            raise shell.ProcessError(13, "myout", "myerr")
```

Comment:
can you use "more unique" strings for myout/myerr, to make sure the check is not false positived?

like, maybe "my special output string".